src/coord/src/coord/arrangement_state.rs

```diff
@@ -171,6 +171,30 @@ impl<T: Timestamp + Copy> Frontiers<T> {
     }
 }
 
+pub struct SinkWrites<T: Timestamp> {
```

Comment:
document everything about this more. also this clearly isn't part of "arrangement state" so either rename file or move elsewhere

Reply:
documented, havent decided on move vs rename file